hazelcast/src/main/resources/hazelcast-default.xml

```diff
@@ -257,10 +257,10 @@
     <!--
         Configuration for a device, which a tiered-store can reference and use for its disk-tier.
     -->
-    <local-device name="default">
+    <local-device name="default-tiered-store-device">
```

Comment:
Shouldn't we add the default capacity here as well?

Reply:
Thanks for the catch, yes we should. Addressed in https://github.com/hazelcast/hazelcast/pull/20523/commits/b9cff34b927f7276da3075d72016d7afabc5a7df